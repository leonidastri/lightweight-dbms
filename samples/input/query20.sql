select Boats.D, Boats.E, Sailors.C, Reserves.G, Sailors.A from Sailors, Boats, Reserves where Sailors.A = Reserves.G AND Boats.D > Reserves.H AND Sailors.A = Boats.E ORDER BY Reserves.G;