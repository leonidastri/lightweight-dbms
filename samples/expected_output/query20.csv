103,1,50,1,1
103,1,50,1,1
107,2,200,2,2